lib/src/models/utils/mapper.dart

```diff
@@ -1,3 +1,7 @@
+import 'dart:convert';
```

Comment:
Not sure if you are happy with dealing with json strings in this class? However, the `QMapper` appeared to be the best place for this logic.